type.go

```diff
@@ -188,9 +188,15 @@ type Type interface {
 	// encoding methods.
 	Decode(dst encoding.Values, src []byte, enc encoding.Encoding) (encoding.Values, error)
 
-	// Assigns a Parquet value to a Go value. Returns an error if conversion is
-	// not possible.
+	// Assigns a Parquet value to a Go value. Returns an error if assignment is
+	// not possible. It is expected that Value is using an expected logical type
+	// which can be done using ConvertValue.
 	AssignValue(dst reflect.Value, src Value) error
+
+	// Convert a Parquet value of the given logical type into a Parquet value
+	// of expected logical type. The converted Value is suitable to be passed to
+	// AssignValue.
```

Comment:
Could we make it more explicit that the method receiver is the type to convert _to_?

Reply:
👍 Sure thing. Addressed in 8f31958